pkg/api/k8s_versions.go

```diff
@@ -63,7 +63,7 @@ var k8sComponentVersions = map[string]map[string]string{
 		"aci-connector":                    "virtual-kubelet:latest",
 		ContainerMonitoringAddonName:       "oms:ciprod01092019",
 		AzureCNINetworkMonitoringAddonName: "networkmonitor:v0.0.4",
-		"cluster-autoscaler":               "cluster-autoscaler:v1.12.1",
+		"cluster-autoscaler":               "cluster-autoscaler:v1.12.2",
```

Comment:
Can we add this to the `install-dependencies.sh` file ? More pre-pulling goodness !